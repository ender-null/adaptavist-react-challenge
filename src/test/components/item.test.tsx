import React from "react";
import { render } from "@testing-library/react";
import Item from "../../components/item";

describe("App", () => {
  const props = {
    word: "test",
    count: 3,
    rarity: 5,
  };
  test("should render the component", () => {
    const { asFragment } = render(<Item {...props} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
