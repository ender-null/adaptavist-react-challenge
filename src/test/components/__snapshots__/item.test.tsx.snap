// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the component 1`] = `
<DocumentFragment>
  <li>
    <span
      class="word"
    >
      test
    </span>
    <span
      class="count"
      style="background: rgb(153, 153, 153);"
    >
      x3
    </span>
  </li>
</DocumentFragment>
`;
